tools/flang1/flang1exe/symacc.h

```diff
@@ -16,7 +16,7 @@ struct SYM;
 #include "sharedefs.h"
 
 /* Keep in sync with INIT_NAMES_SIZE, in syminidf.h. */
-#define STB_NSIZE 5063
+#define STB_NSIZE 5069
```

Comment:
I'm feeling lost, where is `syminidf.h`? I just wanted to make sure that this shouldn't it be guarded by `#ifdef TARGET_SUPPORTS_QUADFP`

Reply:
Two versions of `syminidf.h` are generated from `tools/flang1/utils/symtab/symtab.n` and `tools/flang2/utils/symtab/symtab.n`, respectively. `INIT_NAMES_SIZE` refers to the size of the name string table in the respective `STB` structure used in `flang1` and `flang2`. This size changes as more symbols are added, or when a symbol's name changes. The one in `flang2` is smaller than `flang1`, but some attempt to reuse more code between `flang1` and `flang2` had led to a requirement that the tables be kept the same size. The `symtab.n` files do not contain or support any macro, so this size is the same whether or not the feature is enabled.